android-components/components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/FxaAccountManager.kt

```diff
@@ -774,11 +749,6 @@ open class FxaAccountManager(
         }
     }
 
-    private suspend fun finalizeDevice(authType: AuthType) = account.deviceConstellation().finalizeDevice(
```

Comment:
commented above, looks like a refactoring for readability, not for separating the logic